Week3/MAKEME.md

```diff
@@ -54,59 +73,126 @@ let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
 2\.9 Now if unlike Jim, you don't like meerkats and you want to delete it from the array, but you don't know the position or the `index` of the item in the array, how can you find it?   
 2\.10 Log the index of meerkat to the console. Add a message so it says: "The item you are looking for is at index: " (here you should show the index of the item)
 
-## Step 4: Custom DOM manipulation challenge :mortar_board:
+## Step 6: Custom DOM manipulation challenge :mortar_board:
```

Comment:
This step is not about DOM manipulation. Moved to Week 4?

Reply:
yes, the actual assignment is moved, but i forgot to move the header 🙂